libpod/pod.go

```diff
@@ -6,6 +6,11 @@ import (
 	"github.com/containers/storage"
 )
 
+var (
+	// KernelNamespaces is a list of the kernel namespaces a pod can share
+	KernelNamespaces = []string{"ipc", "net", "pid", "user", "mnt", "uts", "cgroup"}
```

Comment:
Can we use libpod.LinuxNS for this?

Reply:
We can, but it's pretty much just used to print out the namespaces on error in pod_create. So really it's not even very useful.